core/modules/simpletest/tests/common.test

```diff
@@ -2315,6 +2315,7 @@ class CommonValidUrlUnitTestCase extends BackdropUnitTestCase {
   function testValidAbsolute() {
     $url_schemes = array('http', 'https', 'ftp');
     $valid_absolute_urls = array(
+      /* cspell:disable */
```

Comment:
Aren't our inline comments supposed to be in the form `// cspell:disable` ... :thinking: But then again, phpcs would nag because of lowercase start and missing punctuation.

Is this the recommended way to prevent cspell from parsing? (Wasn't involved in any cspell discussions so far, so I'm guessing, yes, but don't know.)

Reply:
Looks like cspell is fairly liberal with the available syntaxes. https://cspell.org/configuration/document-settings/#enable--disable-checking-sections-of-code

I don't have a particularly strong preference personally. Though it does seem like if we don't have a good reason we should stick with the `//` style code comments.

Reply:
I pushed a commit to convert these to `//` and it still is ignoring the desired lines.